fixture-generator/Dockerfile

```
@@ -48,10 +48,12 @@ ENV NODE_OPTIONS="--max-old-space-size=8192"
 # Note: bootstrap also builds packages, including compiling .peggy grammar files
 RUN yarn kbn bootstrap --allow-root
 
-# Install tsx and peggy globally
+# Install tsx, typescript, @types/node, and peggy globally
 # tsx: needed for TypeScript transpilation
+# typescript: needed for type checking with tsc
+# @types/node: needed for Node.js type definitions
 # peggy: needed to compile .peggy grammar files to JavaScript
-RUN npm install -g tsx@4.21.0 peggy
+RUN npm install -g tsx@4.21.0 typescript@5.7.2 @types/node peggy
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)